compiler/test/fail_compilation/diag8101.d

```diff
@@ -62,3 +62,6 @@ void main()
     t_1();
     t_2();
 }
+
+// ignored
+deprecated void f_2(char);
```

Comment:
Without this pull, after the `f_2` candidates are printed this line triggers:
```
... (2 more, -v to show) ...
```
Instead of the existing `1 more`.